bazel/api-gen/test/BUILD.bazel

```diff
@@ -4,5 +4,8 @@ generate_api_docs(
     name = "test",
```

Comment:
At some point, we should likely have some actual `test` targets that will ensure this works (also catching issues like the RBE one)